cli/functionary/config.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+import click
+from dotenv import dotenv_values
+
+
+def save_config_value(parameter, value):
```

Comment:
Since we're using dotenv (at least for now), take a look at the [documentation](https://saurabh-kumar.com/python-dotenv/reference/).  I suspect that `set_key` is the thing you want here.